src/optimizer/OptGroup.h

```diff
@@ -79,6 +79,10 @@ class OptGroupNode final {
         return dependencies_;
     }
 
+    void setDeps(std::vector<OptGroup *> deps) {
```

Comment:
where to use it？

Reply:
RemoveNoopProjectRule.cpp:61

Reply:
Why not use `depndsOn`?

Reply:
I want to set them all at once instead of writing a for block.